templates/emr/terraform/modules/emr/variables.tf

```diff
@@ -0,0 +1,32 @@
+##
+## Input Variables
+##
+variable "aws_access_key" {}
+variable "aws_secret_key" {}
+variable "aws_vpc_id" {}
+variable "aws_subnet_id" {}
+
+variable "aws_ssh_public_key" {
+  default = ""
+}
+variable "aws_region" {
+  default = "us-east-1"
```

Comment:
IMHO we should not set default for AWS_REGION. I think that if the user has the `AWS_REGION` env var set, it's expected to use the region from this env var, but it'll use the default value instead. If we don't provide a default it'll fail and the user will be informed that the region needs to be set as terraform variable.